tests/wpt/web-platform-tests/html/semantics/forms/form-submission-0/submit-entity-body.html

```diff
@@ -5,14 +5,46 @@
 <script>
 var simple_tests = [
   {
-    name: "form submission should navigate to url with x-www-form-urlencoded",
+    name: "form submission from form should navigate to url with x-www-form-urlencoded",
     input: "<input name=foo value=bara>",
-    enctype: "application/x-www-form-urlencoded"
+    enctype: "application/x-www-form-urlencoded",
+    submitelement: "",
+    submitaction: function(doc) { doc.getElementById("testform").submit(); }
```

Comment:
I would like to do that, but the form needs to have `.submit()` called, but `input` and `button` need to have `.click()` called.